.github/workflows/rasa-tests.yml

```diff
@@ -0,0 +1,103 @@
+name: Rasa CI
+
+on:
+  # Trigger analysis when pushing in master or pull requests, and when creating
+  # a pull request.
+  push:
+    branches:
+      - rasa_testing
+      - main
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+jobs:
+  type-testing:
+    name: Test Types
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
```

Comment:
Currently this is repeated for each job, which of course also allows them to run in parallel.